### ISSUE TITLE
[Accessibility] Separator component does not have a method to make it decorative

### ISSUE THREAD

**Issue by GrimaceDoesA11y** [adobe/aem-core-wcm-components#2334]
## Bug Report

**Current Behavior**
Screen reader always has focus on Separator component

**Expected behavior/code**
There should be a mechanism to mark the separator component as decorative so that screen reader technology can ignore it. 

**Environment**
- AEM version and patch level (e.g. AEM 6.5)
- Core Components version Separator V1

**Possible Solution**

Suggested solution is to provide an option for authors to toggle the separator component between decorative or non-decorative. 

**Additional context / Screenshots**

**Comment by ky940819** [comment]
Can you confirm my understanding of your requirements?

- You want the author dialog of the Separator component to have a checkbox field labeled (appros) "Separator is decorative" similar in line to how Images can be marked as decrotive?
- When selected, you want `<hr role="none" />` to be the output HTML?

The implication here being that `<hr />` might not be "Thematic changes" in the content.

**Comment by gabrielwalt** [comment]
Created Jira issue SITES-9357
